mail_full_expand/static/src/components/message/mail_full_expand.js

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2014-2015 Grupo ESOC <http://www.grupoesoc.es>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+odoo.define(
+    "mail_full_expand/static/src/components/message/mail_full_expand.js",
+    function (require) {
+        "use strict";
+
+        const components = {
+            Message: require("mail/static/src/components/message/message.js"),
+        };
+        const {patch} = require("web.utils");
+
+        patch(
+            components.Message,
+            "mail_full_expand/static/src/components/message/mail_full_expand.js",
+            {
+                async _onClickMessageFullExpand() {
+                    const action = await this.rpc({
+                        route: "/web/action/load",
+                        params: {action_id: "mail_full_expand.mail_message_action"},
+                    });
+                    action.res_id = this.message.id;
+                    this.trigger("do-action", {action: action});
+                },
```

Comment:
I guess there must be some error around here.

It works while on a record chatter, but it fails when in the discuss app:

https://user-images.githubusercontent.com/973709/140027594-0b7d033f-9e7e-4501-8c7d-810eeac63edf.mp4